PKGBUILD

```diff
@@ -18,14 +18,22 @@ optdepends=('docker: local Docker support'
 source=("portainer.service"
         "portainer.png"
         "portainer.desktop"
-        "portainer-${pkgver}-src.tar.gz::${url}/archive/${pkgver}.tar.gz"
-        "${url}/releases/download/${pkgver}/portainer-${pkgver}-linux-amd64.tar.gz")
+        "portainer-${pkgver}-src.tar.gz::${url}/archive/${pkgver}.tar.gz")
 
 sha256sums=('08603677ac3c01235fcd740892bfae9277bd163b908f62b22e6e7edfdb61976c'
             '8cb50d80f1463cef0a907b7f26ec6387b792182959f51f8cd19dcb6f955b886e'
             '82f7fca2af76e52147397c3b7b07091b72c1be7c82da6bc47e53001306759635'
-            '06ff058dd9d62c9fc9cc2a1ad9fe42d5ca8371ee27f24bdbec276b91539a3fae'
-            'f8dfae3f172dbe5efc92c37a5c0d01df98dabcdc60cc3970cc3045cb8d3b6ec7')
+            '06ff058dd9d62c9fc9cc2a1ad9fe42d5ca8371ee27f24bdbec276b91539a3fae')
+
+source_x86_64=("${url}/releases/download/${pkgver}/portainer-${pkgver}-linux-amd64.tar.gz")
```

Comment:
We'll have to update the `arch=(...)` list on line 10

Once this is done, `make update`/`updpkgsums` will be able to populate each architecture's checksums as normal without any further Makefile changes.

Reply:
It's done